render/egl.c

```diff
@@ -459,7 +459,9 @@ void wlr_egl_save_context(struct wlr_egl_context *context) {
 }
 
 bool wlr_egl_restore_context(struct wlr_egl_context *context) {
-	return eglMakeCurrent(context->display, context->draw_surface,
+	EGLDisplay display = context->display ?
+		context->display : eglGetCurrentDisplay();
```

Comment:
`eglGetCurrentDisplay` can return `EGL_NO_DISPLAY` as well

Reply:
Yes, but this is not an actual use case.

Reply:
Actually, perhaps it's safe to say that `context->display == eglGetCurrentDisplay() == EGL_NO_DISPLAY` means that there was and is no context, so no action needs to be taken.

Reply:
Yeah, LGTM

Reply:
Updated

Reply:
Oh, oops. Need to fix one thing!

Reply:
Fixed now